### ISSUE TITLE
Visibility Problem

### ISSUE THREAD

**Issue by TartejBrothers** [Zolske/notes.zoltankepes.com#1]
1. The site has many dark icons such as the ones in the navbar and in the Body with the GitHub download button, but when changing to dark mode, these don't change to white and makes them very hard to see. 
2. The icons are not horizontally aligned with the text.
3. Hover on light mode is not that prominent.

**Comment by Zolske** [comment]
Yes, you are write, I was not planning to use the dark mode so much.
I am not sure when I will have time to fix it, but thank you bringing it to my attention.
